zerver/actions/message_edit.py

```diff
@@ -628,7 +634,7 @@ def do_update_message(
             topic_only_edit_history_event["prev_stream"] = edit_history_event["prev_stream"]
             topic_only_edit_history_event["stream"] = edit_history_event["stream"]
 
-        messages_list = update_messages_for_topic_edit(
+        later_messages, propagate_changes = update_messages_for_topic_edit(
```

Comment:
I like the effect this has of putting the saves next to each other. Maybe `propagate_changes` => `save_changes_for_propagation_mode` for the same of this little function would be good?